rust/hook-common/src/kafka_messages/app_metrics.rs

```diff
@@ -118,22 +119,25 @@ where
     Ok(category)
 }
 
+impl fmt::Display for ErrorType {
```

Comment:
Good call, `Display` was laziness while getting it working but I agree people shouldn't be afraid to change `Display` freely.

Reply:
Ah, fixtures were working by chance because of a fallback of defaults, which decoded `ConnectionError` (for example) for me. We may as well be robust to this so I went ahead and added them to my deser rather than changing the fixtures.